website/docs/components/form/textarea/partials/accessibility/accessibility.md

```diff
@@ -1,8 +1,25 @@
-#### Known Issues
+## Conformance rating
 
-**Links within labels, help text, or error text:** If a link is used within a label, helper text, or error text, it will not be presented as a link to the user with a screen reader; only the text content is read out. As such, it is generally preferable to avoid links within help/error text or labels; however, we understand that this may not be avoidable in some cases. Please use sparingly until a good known alternative approach is determined.
+### Form::Textarea::Field
 
-#### Applicable WCAG Success Criteria (Reference)
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+`Form::Textarea::Field` is conformant when lorem ipsum
+
+### Form::Textarea::Base
+
+<Doc::Badge @type="warning">Conditionally conformant</Doc::Badge>
+
+`Form::Textarea::Base` is not conformant until lorem ipsum 
+
+!!! Info  
+
+**Links within labels, help text, or error text:** 
+
+If a link is used within a label, helper text, or error text, it will not be presented as a link to the user with a screen reader; only the text content is read out. As such, it is generally preferable to avoid links within help/error text or labels; however, we understand that this may not be avoidable in some cases. Please use sparingly until a good known alternative approach is determined.
```

Comment:
```suggestion
If a link is used within a label, helper text, or error text, it will not be presented as a link to the user with a screen reader; only the text content is read out. It is generally preferable to avoid links within help/error text or labels; however, we understand that this may not be avoidable in some cases. Use sparingly until a good known alternative approach is determined.
```